lib/life_game/hoge.rb

```diff
@@ -2,4 +2,15 @@ class Hoge
   def hello
     puts "hello"
   end
+
+  def hello2
+    puts 'hello2'
+  end
+
+  def hello3
+   puts 'hello2'
+  end
+  def hello3
```

Comment:
Lint/DuplicateMethods: Method `Hoge#hello3` is defined at both lib/life_game/hoge.rb:10 and lib/life_game/hoge.rb:13.